jupyterbook/content/code_gallery/data_management_notebooks/2017-05-14-running_compliance_checker.ipynb

```diff
@@ -31,7 +31,7 @@
      "name": "stdout",
```

Comment:
<p>Looks like this is an error message?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ioos/ioos_code_lab/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Argh. This commit should not be here. I based it of the wrong branch. Let me remove it.</p>